packages/amplify_authenticator/lib/src/widgets/default_forms.dart

```diff
@@ -1,94 +1,107 @@
 import 'package:amplify_authenticator/src/enums/alias.dart';
+import 'package:amplify_authenticator/src/text_customization/auth_strings_resolver.dart';
 import 'package:amplify_authenticator/src/widgets/forms.dart';
 import 'package:amplify_authenticator/src/widgets/form_fields.dart';
+import 'package:flutter/widgets.dart';
 
 /// These default forms will be used as long as there are no
 /// arguments being passed from the main Authenticator wrapper.
 
 // The initial auth flow will be setup based on an username alias
 // that comes from the main Authenticator widget.
 class DefaultForms {
-  static SignInForm signInForm(Alias alias) {
+  late BuildContext _context;
+  void set context(BuildContext context) {
+    _context = context;
+  }
+
+  SignInForm signInForm(Alias alias, AuthStringResolver resolver) {
     late String _title;
     late String _hintText;
     switch (alias) {
       case Alias.username:
-        _title = 'Username*';
-        _hintText = 'Enter your username';
+        _title = resolver.inputs.username_title(_context);
+        _hintText = resolver.inputs.username_hint(_context);
```

Comment:
Nit: Might want to consider extracting `resolver.inputs`. If, for some reason, we ever changed the inputs resolver to `resolver.fields` or something we could just make that change in one place. But this is nit since it doesn't make this look much more concise and ... if we ever did change the name we'd *probably* be inclined to rename the variable anyway 😅  So feel free to ignore, just a minor suggestion to consider

Reply:
Yeah, this is a good catch. I was thinking we'd refine this a bit once the activity on the default_forms file settles down (@israx and I were both heavily modifying at the same time).